<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectroApp</class>
 <widget class="QMainWindow" name="SpectroApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectral Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnDarkRef">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Dark Reference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnWhiteRef">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>White Reference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAttWhiteRef">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Attenuated White Reference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMercuryRef">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Mercury Reference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNeonRef">
        <property name="minimumSize">
         <size>
          <width>198</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Neon Reference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAimingBeam">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Aiming Beam</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="plotArea" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
